apps/core/components/passengerForm/PassengerForm.js

```diff
@@ -94,24 +92,20 @@ class PassengerForm extends React.Component<Props, State> {
     // If is in edit mode preload passenger data into the form state
     if (editModeOpened) {
       if (props.prefillData) {
+        const dateOfBirth =
```

Comment:
maybe you can destructure `const { prefillData } = props.prefillData`, as it's accessed quite a lot

Reply:
fixed, thanks!